Hazel/src/Hazel/Core/EntryPoint.h

```diff
@@ -5,6 +5,10 @@
 
 extern Hazel::Application* Hazel::CreateApplication();
 
+extern "C" {
+	__declspec(dllexport) DWORD NvOptimusEnablement = 0x00000001;
```

Comment:
please use uint32_t instead of DWORD. `__declspec(dllexport)` is MSVC specific, you should use `export`

Reply:
Done